test/specs/rules/disallow-space-after-keywords.js

```diff
@@ -1,7 +1,7 @@
 var Checker = require('../../../lib/checker');
 var assert = require('assert');
 
-describe('rules/disallow-space-after-keywords', function() {
+describe.only('rules/disallow-space-after-keywords', function() {
```

Comment:
I don't think we currently have a way to support disallow `describe.only` together - only `describe` and `.only`. That would be an enhancement?